kie-server-parent/kie-server-tests/kie-server-integ-tests-jbpm/src/test/filtered-resources/kjars-sources/query-definition-project/pom.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.kie.server.testing</groupId>
+    <artifactId>common-parent</artifactId>
+    <version>1.0.0.Final</version>
+  </parent>
+
+  <artifactId>query-definition-project</artifactId>
+  <version>1.0.0.Final</version>
+  <name>query-definition-project</name>
+
+  <dependencies>
+    <dependency>
+      <groupId>org.jbpm</groupId>
+      <artifactId>jbpm-services-api</artifactId>
+      <version>${version.org.kie}</version>
+      <scope>system</scope>
+      <systemPath>${user.home}/.m2/repository/org/jbpm/jbpm-services-api/${version.org.kie}/jbpm-services-api-${version.org.kie}.jar</systemPath>
```

Comment:
This will likely fail when the local maven repository is somewhere else that ~/.m2 (this is true for the Jenkins PR builds). I will think about how we could do this better (no good ideas off the top of my head).